data_steward/utils/pipeline_logging.py

```diff
@@ -111,3 +159,86 @@ def setup_logger(log_filepath_list, console_logging=True):
     for filename in log_path:
         log_list.append(create_logger(filename, console_logging))
     return log_list
+
+
+def _get_date_str():
+    """
+    Get current date formatted using FILENAME_FMT
+    :return: 
+    """
+    return datetime.today().strftime(FILENAME_FMT)
+
+
+def _get_log_file_path():
+    """
+    Get the abs path of the log file to use. 
+    
+    The location is DEFAULT_LOG_DIR and the file name is the current
+    date formatted using FILENAME_FMT.
+
+    :return: absolute path to the log file
+    """
+    date_str = _get_date_str()
+    return os.path.join(DEFAULT_LOG_DIR, f'{date_str}.log')
+
+
+def _get_config(filename, level):
+    """
+    Get a dictionary which describes the logging configuration
+    
+    :param filename: Create the FileHandler using the specified filename.
+    :param level: Set the root logger level to the specified level 
+                  (i.e. logging.{DEBUG,INFO,WARNING,ERROR}).
+    :return: the default configuration dict
+    """
+    # copy _DEFAULT_CONFIG to avoid modifying it
+    default_config = dict(_DEFAULT_CONFIG)
+    default_config['root']['level'] = level
+    file_log_handler_dict = default_config['handlers'][_FILE_HANDLER]
+    file_log_handler_dict['filename'] = filename
+    return default_config
+
+
+def _except_hook(exc_type, exc_value, exc_traceback):
+    """
+    Log exception info to root logger. Used as a hook for uncaught exceptions prior 
+    to system exit.
+    
+    :param exc_type: type of the exception
+    :param exc_value: the exception
+    :param exc_traceback: the traceback associated with the exception
+    """
+    root_logger = logging.getLogger()
+    root_logger.critical('Uncaught exception',
+                         exc_info=(exc_type, exc_value, exc_traceback))
+
+
+def configure(level=logging.INFO):
```

Comment:
This assumes the user always wants a stream handler.  We shouldn't make that assumption.  Processes that are automatically started won't want to add a stream handler, just a file handler.  We should allow passing a flag into to the configure function to determine whether or not to add a stream handler.

```suggestion
def configure(level=logging.INFO, console_log=False):
```